File: UserDomain/ValidateCodeService/Controller/validateCodeController.js
const validate_code = async function(req,res){
    let code = req.params['code'];
    let email = req.params['email'];

    let user = await User.findOne({email:email});

    if(code == user.code_reset){
        res.status(200).send({data:true});
    }else{
        res.status(200).send({data:false});
    }

}


module.exports = {
    validate_code
}

File: UserDomain/ValidateCodeService/controller/validate_code_controller.py
from fastapi import HTTPException
from models.user import users_collection

async def validate_code(email: str, code: str):
    user = await users_collection.find_one({"email": email})
    if not user:
        raise HTTPException(status_code=404, detail="User not found")

    return {"data": code == user["code_reset"]}
